exec/cnex/cell.c

```diff
@@ -113,36 +127,60 @@ Cell *cell_fromCell(const Cell *c)
             }
             x->boolean = FALSE;
             x->number = number_from_uint32(0);
+            x->object = NULL;
             x->string = NULL;
             x->address = NULL;
             x->dictionary = NULL;
             break;
         case cBoolean:
             x->boolean = c->boolean;
             x->number = number_from_uint32(0);
+            x->object = NULL;
             x->string = NULL;
             x->address = NULL;
             x->array = NULL;
             x->dictionary = NULL;
             break;
+        case cBytes:
+            x->string = string_copyString(c->string);
+            x->boolean = FALSE;
+            x->number = number_from_uint32(0);
+            x->object = NULL;
+            x->address = NULL;
+            x->array = NULL;
+            x->dictionary = NULL;
+            break;
         case cDictionary:
             x->dictionary = dictionary_copyDictionary(c->dictionary);
             x->boolean = c->boolean;
```

Comment:
I thought you already fixed this to assign `FALSE`.

Reply:
It is fixed now.